Sources/RestBird/Request/MultipartRequest.swift

```diff
@@ -9,7 +9,7 @@ import Foundation
 
 public enum Multipart {
     case path(url: URL, name: String)
-    case data(data: Data, name: String, mimeType: String)
```

Comment:
I would still leave this case and handle it also in AlamofireSessionManager, just in case someone doesn't want to use fileName.